cmd/auth/login.go

```diff
@@ -20,6 +36,23 @@ var loginCmd = &cobra.Command{
 		defer perisistentAuth.Close()
 		ctx, cancel := context.WithTimeout(cmd.Context(), loginTimeout)
 		defer cancel()
+
+		profileFlag := cmd.Flag("profile")
+		if profileFlag != nil && profileFlag.Value.String() != "" {
+			perisistentAuth.Profile = profileFlag.Value.String()
+		} else {
+			profiles, err := getProfilesMap(cmd.Context())
+			if err != nil {
+				return err
+			}
+			if len(profiles) > 0 {
+				profile, err := cmdio.Select(ctx, profiles, "~/.databrickscfg profile")
```

Comment:
This shouldn't be a selector but rather a regular input box. Users can specify a new profile name that isn't yet present in the databrickscfg file.